src/managedccfs/azext_managedccfs/aaz/latest/confidentialledger/managedccfs/_update.py

```diff
@@ -0,0 +1,267 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "confidentialledger managedccfs update",
+    is_preview=True,
+)
+class Update(AAZCommand):
+    """Patch a Managed CCF instance.
+    """
+
+    _aaz_info = {
+        "version": "2023-01-26-preview",
+        "resources": [
+            ["mgmt-plane", "/subscriptions/{}/resourcegroups/{}/providers/microsoft.confidentialledger/managedccfs/{}", "2023-01-26-preview"],
+        ]
+    }
+
+    AZ_SUPPORT_NO_WAIT = True
+
+    def _handler(self, command_args):
+        super()._handler(command_args)
+        return self.build_lro_poller(self._execute_operations, None)
+
+    _args_schema = None
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        if cls._args_schema is not None:
+            return cls._args_schema
+        cls._args_schema = super()._build_arguments_schema(*args, **kwargs)
+
+        # define Arg Group ""
+
+        _args_schema = cls._args_schema
+        _args_schema.name = AAZStrArg(
+            options=["-n", "--name"],
+            help="A unique name for the instance.",
+            required=True,
+            id_part="name",
+            fmt=AAZStrArgFormat(
+                pattern="^[^-0-9][A-Za-z0-9-]{1,33}[A-Za-z0-9]$",
+            ),
+        )
+        _args_schema.resource_group = AAZResourceGroupNameArg(
+            required=True,
+        )
+
+        # define Arg Group "ManagedCCF"
+
+        _args_schema = cls._args_schema
+        _args_schema.tags = AAZDictArg(
+            options=["--tags"],
+            arg_group="ManagedCCF",
+            help="Resource tags.",
+        )
+
+        tags = cls._args_schema.tags
+        tags.Element = AAZStrArg()
+
+        # define Arg Group "Properties"
+
+        _args_schema = cls._args_schema
+        _args_schema.deployment_type = AAZObjectArg(
+            options=["--deployment-type"],
+            arg_group="Properties",
+            help="Deployment Type of Managed CCF",
+        )
+        _args_schema.member_identity_certificates = AAZListArg(
+            options=["--member-identity-certificates"],
+            arg_group="Properties",
+            help="List of member identity certificates for  Managed CCF",
+        )
+        _args_schema.node_count = AAZIntArg(
+            options=["--node-count"],
+            arg_group="Properties",
+            help={"short-summary": "Number of CCF nodes in the instance.", "long-summary": "If the argument is omitted, a default value of 3 is used. The maximum supported size is 9 nodes."},
+            is_preview=True,
+            default=3,
```

Comment:
Update commands should not have parameters with default values. Let's change it in the workspace and re-generate.